### ISSUE TITLE
Error when creating separator

### ISSUE THREAD

**Issue by matiboy** [TylorS/typed-prompts#4]
**Code to reproduce the issue:**
```
import { prompt, list, separator } from 'typed-prompts';

prompt([
  list('das','dsa',[
    'Hi!',
    separator()
  ])
]);
```

**Expected behavior:**
Creates a new separator. In `index.ts`, replacing `createSeparator` with `return new Separator();` seems to work

**Actual behavior:**
`Error: inquirer_1.separator is not a function`

**Versions of packages used:**
- `typed-prompts=1.1.0`
- `inquirer=1.2.3`

**Comment by TylorS** [comment]
Ahh yes, this is definitely a bug. I'm not going to be ale to fix it tonight. PR welcomed of course. Otherwise I can get to this tomorrow.

**Comment by TylorS** [comment]
Merged your PR and released in 1.2.0. Thanks again!
